2_process/src/process_helpers.R

```diff
@@ -14,6 +14,10 @@ extract_morphometry <- function(config_fn) {
   return(morphometry_out)
 }
 
+nml_to_morphometry <- function(in_ind) {
+  purrr::map(readRDS(as_data_file(in_ind)), `[`, c("latitude", "longitude", "H", "A"))
```

Comment:
just a check here to verify your functions that are related to morphometry, H/A still work the same way now that we've adjusted those all to have the elevations of the lakes taken into account (instead of all being 320). I'm 99% certain you are fine in all of these cases, but asking here in case you think it may be an issue.

Reply:
Thanks for raising this point. I have always just used H/A and then [converted later to hypso using max H](https://github.com/USGS-R/lake-temperature-out/blob/main/3_summarize/src/annual_thermal_metrics.R#L46-L49). I don't think that change impacts my uses here.